app/src/main/java/com/framgia/soundclound/screen/detailgenre/GenreDetailViewModel.java

```diff
@@ -3,14 +3,18 @@
 import android.content.Context;
 import android.databinding.BaseObservable;
 import android.databinding.Bindable;
+import android.databinding.ObservableField;
 import android.support.v7.app.AppCompatActivity;
+import android.view.ScaleGestureDetector;
```

Comment:
ERROR: Unused import: android.view.ScaleGestureDetector. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports